Codeinwp/templates-patterns-collection
# fix: welcome notice redirects to old obd

### Summary
- Trigger the new onboarding if it exists when clicking the welcome notice from Neve

### Test instructions
- Install neve
- Have this version of TPC on your instance but don't activate it
- Click on "Try one of our ready..." button from the welcome notice


Closes #286.
